lib/config/presets/internal/regex-managers.ts

```diff
@@ -19,9 +19,9 @@ export const presets: Record<string, Preset> = {
       'Update `_VERSION` environment variables in GitHub Action files.',
     regexManagers: [
       {
-        fileMatch: ['^.github/workflows/[^\\.]+\\.ya?ml$'],
+        fileMatch: ['^.github/workflows/[^/]+\\.ya?ml$'],
         matchStrings: [
-          '# renovate: datasource=(?<datasource>[a-z-]+?) depName=(?<depName>[^\\s]+?)(?: (lookupName|packageName)=(?<packageName>[^\\s]+?))?(?: versioning=(?<versioning>[a-z-0-9]+?))?\\s  .+?_VERSION\\s*:\\s*["\']?(?<currentValue>.+?)["\']?\\s',
+          '# renovate: datasource=(?<datasource>[a-z-]+?) depName=(?<depName>[^\\s]+?)(?: (?:lookupName|packageName)=(?<packageName>[^\\s]+?))?(?: versioning=(?<versioning>[a-z-0-9]+?))?\\s+\\S+?_VERSION\\s*:\\s*["\']?(?<currentValue>.+?)["\']?\\s',
```

Comment:
```suggestion
          '# renovate: datasource=(?<datasource>[a-z-]+?) depName=(?<depName>[^\\s]+?)(?: (?:lookupName|packageName)=(?<packageName>[^\\s]+?))?(?: versioning=(?<versioning>[a-z-0-9]+?))?\\s+[A-Za-z0-9_]+?_VERSION\\s*:\\s*["\']?(?<currentValue>.+?)["\']?\\s',
```

should we restrict further, as special chars doesn't make sense there?

Reply:
sure, fine by me :)